### PR TITLE
Fix compatibility with PHP 5.4

### LINKED ISSUE #22960: Admin page down after update to 9.0
After update to 9.0 the admin page are down.
Only an white page will show.
OS: CentOS7
Log(journal):
 {PHP} Can't use method return value in write context at /var/www/html/owncloud/apps/files_external/lib/google.php#268

### PR DESCRIPTION
**Pull request by LukasReschke**
Fixes https://github.com/owncloud/core/issues/22960

@karlitschek We should backport this into 9.0.1

### CODE REVIEW

**Comment by LukasReschke** [comment]
Confirmed working by reporter (@tuxmaster) in https://github.com/owncloud/core/issues/22960#issuecomment-193873670

**Comment by Xenopathic** [comment]
:+1:

**Comment by karlitschek** [comment]
please backport :+1:

**Comment by rullzer** [comment]
:+1:

**Comment by LukasReschke** [comment]
stable9 is at https://github.com/owncloud/core/pull/22965

File: apps/files_external/lib/google.php [hunk 1]
```diff
@@ -265,7 +265,8 @@ public function opendir($path) {
 				foreach ($children->getItems() as $child) {
 					$name = $child->getTitle();
 					// Check if this is a Google Doc i.e. no extension in name
-					if (empty($child->getFileExtension())
+					$extension = $child->getFileExtension();
+					if (empty($extension)
```
Comment (PVince81) [inline apps/files_external/lib/google.php]: ah....  classic

**Comment by PVince81** [comment]
:+1:

**Pull request merged**